tests/test_converter.py

```diff
@@ -454,6 +458,140 @@ def test_strikeout_word():
     assert newline_lf(markdown_output) == expected_markdown
 
 
+def test_annotated_spaces():
+    input = {
+        'object': 'block',
```

Comment:
Is it necessary to include all of this meta data, e.g., `created_time`, in the test fixture here? It seems unrelated to what we're testing. If it is needed, perhaps there's some way we can create a mock utility so that we can hide this complexity outside of the tests. If other tests include this metadata unnecessarily, I suggest we refactor those other tests as well.

Reply:
no, I can certainly and will delete some of it